### PR TITLE
[Olly] [AWS] Update Billing Overview dashboard

### PR DESCRIPTION
**Pull request by milan-elastic**
- Bug

## What does this PR do?

- Update visualization aggregation from sum to last_value
- Currently, all visualization has sum aggregation which was leading to this [issue](https://github.com/elastic/obs-infraobs-team/issues/1181)

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Closes https://github.com/elastic/obs-infraobs-team/issues/1181

## Screenshots

Before changes using sum aggregation with 12h period of time

image

After changes using last_value aggregation with 12h period of time

image

### CODE REVIEW

**Review by harnish-elastic — APPROVED** [review]

**Comment by agithomas** [comment]
@milan-elastic , please enable all items in the `checklist` in the description.

**Comment by agithomas** [comment]
As the change is related to a bug, please elaborate on what is the issue for which the fix is applied in the changelog.

**Comment by milan-elastic** [comment]
> As the change is related to a bug, please elaborate on what is the issue for which the fix is applied in the changelog.

Done

File: packages/aws/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.2.2"
+  changes:
+    - description: Fix AWS Billing Overview Dashboard aggregation from "sum" to "last_value".
```
Comment (agithomas) [inline packages/aws/changelog.yml:4]:
```suggestion
    - description: Fix incorrect value displayed under Total Estimated Charges under AWS Billing Overview Dashboard.
```

File: packages/aws/kibana/dashboard/aws-e6776b10-1534-11ea-841c-01bf20a6c8ba.json [hunk 2]
```diff
@@ -87,16 +87,16 @@
                                                     },
                                                     "isBucketed": false,
                                                     "label": "Total Estimated Charges ($)",
-                                                    "operationType": "sum",
+                                                    "operationType": "last_value",
```
Comment (agithomas) [inline packages/aws/kibana/dashboard/aws-e6776b10-1534-11ea-841c-01bf20a6c8ba.json:90]: I see that other panels in the dashboard use `avg` and this panel uses `last_value`. Please verify the correctness of other panels

**Review by agithomas — COMMENTED** [review]
Review comments added.

**Review by kaiyan-sheng — APPROVED** [review]

**Review by drewdaemon — APPROVED** [review]
This feels like the right fix 👍

**Review by agithomas — APPROVED** [review]
LGTM!

**Pull request merged**